src/components/PhoneNumber.tsx

```diff
@@ -1,27 +1,45 @@
 import {useState} from "react";
 import "./phoneNumber.scss";
+import PhoneInput from 'react-phone-input-2';
```

Comment:
There is no npm dependancy 'react-phone-input-2'. 

You need to add a package.json with this dependancy into this merge request